js/src/threepio.js

```diff
@@ -10,39 +11,34 @@ export default class Threepio {
     this.tf = tf;
   }
 
-  normalizeFunctionName(name) {
-    return name
+  _normalizeFunctionName(name, toLang) {
+    const result = name
       .match(NORMALIZATION_REGEX)
       .join('')
       .toLowerCase();
-  }
 
-  matchArgs(args, argMap) {
-    // Sort args by index
-    argMap.args.sort((a, b) =>
-      a.index === b.index ? 0 : a.index < b.index ? -1 : 1
-    );
-    for (const [i, arg] of args.entries()) {
-      // TODO: match args based on fromLibrary name
-      // TODO: Ensure order is preserved
-      // TODO: Ensure there are not any missing required arguments
-      console.log(i, arg);
-      continue;
+    if (result in this.mappedCommands[toLang]) {
```

Comment:
Hmm, does it expect function names to be same in both frameworks?
E.g. func name in torch must directly match to func name in tfjs?
I imagined mapping will take in consideration source and target frameworks, like
```
torchCmdInfo = mappedCommands['torch'][name] // source framework
tfjsName = torchCmdInfo.how_that_will_be_in['tfjs']
tfjsCmdInfo = mappedCommands['tfjs'][tfjsName] // target framework
// ... args mapping ...
```

Reply:
We now store the function name mapping in our static json.